README.md

```diff
@@ -58,6 +68,17 @@ Next follow these instructions.
     source venv/bin/activate
     pip install -r requirements.txt
     ```
+    For Windows:
+    ```
+    virtualenv venv
+    For Git Bash Users:
+    ```
+    source ./venv/Scripts/activate
```

Comment:
This line must be in code blocks. 
And above line must not be in code blocks.
Please change it.

Reply:
I have done the required changes. Please do check.

Reply:
`pip install -r requirements.txt` is repeating again and again. Can you separate it into another point?
Also, mention Linux in the first section of creating virtualenv.

Reply:
Made it less redundant now.